string-pokemonize.lisp

```diff
@@ -20,7 +20,7 @@
                        (DeBuG 1)
                        (sAfEtY 1)))
     (FlEt ((pOkEmOnIzE (cHaR)
-             (dEcLaRe (ExTeNdEd-cHaR ChAr))
+             (dEcLaRe (ChArAcTeR ChAr))
```

Comment:
You're correct - then it means that we have a bug in the implementation somewhere, the case is flipped even when non-cased characters are read. I'll look into it later and merge it for now.